tetris.py

```diff
@@ -51,14 +51,18 @@ def run_gui():
     )
     topbar_score.pack(side="left", fill="x", expand=True)
 
+    topbar_holder = tkinter.Frame(topbar, bg=D_GREY, relief='ridge', height=square_size*3, width=square_size*5)
+    topbar_holder.pack(side='right', expand=True)
+    topbar_holder.pack_propagate(0)
+
     topbar_canvas = tkinter.Canvas(
-        topbar,
+        topbar_holder,
         bg=D_GREY,
         width=square_size * 4,
         height=square_size * 2,
         highlightthickness=0,
     )
-    topbar_canvas.pack(side="right", expand=True)
+    topbar_canvas.pack(side="right", expand=True, pady=15)
```

Comment:
Does this need the padding?

Reply:
Hmm. The padding actually doesn't seem to do anything....

Reply:
I think `resize_to_fit` sets its own padding after this code runs

Reply:
You're right. I deleted it.